AdyenActions/Components/Voucher/VoucherComponent.swift

```diff
@@ -76,17 +94,32 @@ public final class VoucherComponent: AnyVoucherActionHandler {
             let presentableComponent = PresentableComponentWrapper(component: self, viewController: viewController)
             presentationDelegate.present(component: presentableComponent)
         } else {
-            let message = "PresentationDelegate is nil. Provide a presentation delegate to VoucherAction."
+            let message = "PresentationDelegate is nil. Provide a presentation delegate to VoucherComponent."
```

Comment:
I have idea to new PR - inject `presentationDelegate` on init and make it non-optional 😆

Reply:
its weak, right? you cannot make it non-optional

Reply:
In this case it doesn't have to be, right?
Conceptually it is not a `delegate` any more, but rather en utility - `VoucherComponent` could not function properly without it (like `AnyAPIClient`)

Reply:
its weak because most probably the delegate retains the delegator, so it has to be weak to prevent retain cycles.

what do you mean it doesn't have to? of course it has to :-)